manifests/r/RealVNC/VNCServer/6.9.0.46599/RealVNC.VNCServer.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RealVNC.VNCServer
+PackageVersion: 6.9.0.46599
+PackageLocale: en-US
+Publisher: RealVNC Ltd
```

```suggestion
Publisher: RealVNC
```
Needs to match ARP (apparently not cough before)

---

Ah - they changed it 😄

---

Oh good, then this will help to match previous and newer versions